plugins/rbac/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@janus-idp/backstage-plugin-rbac",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "tsc": "tsc",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.13.3",
+    "@backstage/core-plugin-api": "^1.5.3",
+    "@backstage/theme": "^0.4.1",
+    "@material-ui/core": "^4.9.13",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.45",
+    "react-use": "^17.2.4"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1 || ^17.0.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.22.9",
+    "@backstage/core-app-api": "^1.9.0",
+    "@backstage/dev-utils": "1.0.17",
+    "@backstage/test-utils": "^1.4.1",
+    "@testing-library/jest-dom": "^5.10.1",
+    "@testing-library/react": "^12.1.3",
+    "@testing-library/user-event": "^14.0.0",
+    "@types/node": "*",
+    "msw": "^1.0.0"
```

I'm sorry, I've just updated the repo to have Backstage compatibility for 1.19.0 (https://github.com/janus-idp/backstage-plugins/pull/858). In order to keep our dependencies in line, can you please rebase and update these to:

```suggestion
  "dependencies": {
    "@backstage/core-components": "^0.13.6",
    "@backstage/core-plugin-api": "^1.7.0",
    "@backstage/theme": "^0.4.3",
    "@material-ui/core": "^4.9.13",
    "@material-ui/icons": "^4.11.3",
    "@material-ui/lab": "^4.0.0-alpha.45",
    "react-use": "^17.4.0"
  },
  "peerDependencies": {
    "react": "^16.13.1 || ^17.0.0"
  },
  "devDependencies": {
    "@backstage/cli": "0.23.0",
    "@backstage/core-app-api": "1.11.0",
    "@backstage/dev-utils": "1.0.22",
    "@backstage/test-utils": "1.4.4",
    "@testing-library/jest-dom": "5.17.0",
    "@testing-library/react": "12.1.5",
    "@testing-library/user-event": "14.5.1",
    "@types/node": "18.18.5",
    "msw": "1.3.2"
```

---

Yeah just saw that the PR got merged. No issues as anyways I was going to update the dependencies. :)